{{cookiecutter.project_name}}/server/settings/environments/development.py

```diff
@@ -128,3 +132,6 @@ def _custom_show_toolbar(request):
         'field-choices-constraint',
     ],
 }
+
+# Disable persistent DB connections
```

Comment:
Let's also put a link to this doc: https://docs.djangoproject.com/en/2.2/ref/databases/#caveats